pkg/cmd/config.go

```diff
@@ -91,15 +91,16 @@ type ConfigFile struct {
 	WorkingTreeAbsPath chezmoi.AbsPath                 `mapstructure:"workingTree"`
 
 	// Password manager configurations.
-	Bitwarden   bitwardenConfig   `mapstructure:"bitwarden"`
-	Gopass      gopassConfig      `mapstructure:"gopass"`
-	Keepassxc   keepassxcConfig   `mapstructure:"keepassxc"`
-	Keeper      keeperConfig      `mapstructure:"keeper"`
-	Lastpass    lastpassConfig    `mapstructure:"lastpass"`
-	Onepassword onepasswordConfig `mapstructure:"onepassword"`
-	Pass        passConfig        `mapstructure:"pass"`
-	Secret      secretConfig      `mapstructure:"secret"`
-	Vault       vaultConfig       `mapstructure:"vault"`
+	AwsSecretsManager awsSecretsManagerConfig `mapstructure:"awsSecretsManager"`
```

Comment:
This is the only change here.

Reply:
Tiny nit: please rename this field to `AWSSecretManager` (https://github.com/golang/go/wiki/CodeReviewComments#initialisms).